docs/appendix/zowe-glossary.md

```diff
@@ -0,0 +1,300 @@
+# Glossary
+
+This glossary is part of a growing list of Zowe terminology.
+
+## Community
+
+* **Open Mainframe Project (OMP)**
+  
+  It is an organization which hosts and promotes development of open source software for the benefit of the IBM z mainframe community, including but not limited to z/OS. Zowe(.org) is one of several programs in this project. See [Open Mainframe Project website](https://www.openmainframeproject.org/) for more information.
+
+* **Zowe Conformance Program** 
+
+  The Zowe Support Provider Conformance Program gives vendors the ability to showcase their Zowe support competencies via well defined criteria. It is administered by the Linux Foundation and Open Mainframe Project.
+ 
+* **Squad** 
+  
+  A squad is a group of people contributing and participating in the Zowe project. Such a group owns one or more projects. Every squad is required to have a representative on the Technical Steering Committee (TSC), and participate in relevant working groups. See [Squads](https://github.com/Zowe/community/blob/master/Technical-Steering-Committee/squads.md) for more information.
+
+* **Zowe Component**
+ 
+  Zowe is a collection of both client and server code. You can install just some of Zowe, or all of it, depending on your needs. Zowe splits the major sections of the code into "Components" where each serve an important purpose. Server Components are packaged in a standardized way to include all services and plugins in one deliverable. Extensions to Zowe can also be delivered as third party Server Components. See: https://docs.Zowe.org/stable/extend/packaging-zos-extensions/#Zowe-component-manifest
+
+* **Plugin**
+
+  A more general term used to describe a modular piece of some component. Depending on component or squad context, a plugin is sometimes referred to as an "app", "extension", "plug-in" etc. A component may have multiple plugins, sometimes working together to form a single purpose or user experience, but an individual plugin belongs to a single component.
+
+* **Extension**
+
+  An extension is generally used to describe additional, non-default Zowe plugins or components.
+
+* **Core components**
+
+  The definition of a core component is governed by the TSC but a core component usually means a packaged, foundational piece as part of base Zowe. From the perspective of a conformant support provider, providing support for Zowe refers to providing support for each core component of Zowe though a provider may put their own limitations on what they support. A core component is usually actively maintained by one or more squads. A component has a component manifest file, that helps identify it with the rest of Zowe. See: https://docs.zowe.org/stable/appendix/server-component-manifest/
+
+## Core
```

Comment:
Is there any reason to use other descriptions than the ones on zowe.org?

Reply:
I don't know the answer to this, but can you provide a link to the zowe.org description?

Reply:
Probably resolved by the updates I made.